### PR TITLE
Document bulk loading AWS keys

### LINKED ISSUE #124: Document AWS bulk loading for eth2
Document bulk retrieval of AWS signing keys for Eth2.

See:
https://github.com/ConsenSys/web3signer/pull/589
https://github.com/ConsenSys/web3signer/issues/499

### PR DESCRIPTION
**Pull request by alexandratran**
# Pull request checklist

Use the following template to make sure your PR fits the Web3Signer documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-locally).

## Describe the change

Document CLI options and how to bulk load keys from AWS secrets manager.

## Issue fixed

fixes #124 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [ ] You've [previewed your changes on Read the Docs](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-on-read-the-docs)
  and added a [preview link](#preview).

### CODE REVIEW

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 1]
```diff
@@ -84,6 +84,247 @@ When
...
+=== "Syntax"
+
+    ```bash
+    --aws-secrets-auth-mode=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --aws-secrets-enabled=ENVIRONMENT
```
Comment (usmansaleem) [inline docs/Reference/CLI/CLI-Subcommands.md:128]: Should be `--aws-secrets-auth-mode`

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 2]
```diff
@@ -84,6 +84,247 @@ When
...
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    eth2.aws-secrets-tag-values-filter: ["tagValue1","tagValue2"]
+    ```
+
+Optional comma-separated list of tag values filter to apply while fetching secrets from AWS Secrets
+Manager.
```
Comment (usmansaleem) [inline docs/Reference/CLI/CLI-Subcommands.md:326]: At the moment, these options are written in alphabetical order (which makes sense). For ease of readability for end user, can we re-order them as:
~~~
--aws-secrets-enabled
--aws-secrets-auth-mode
~~~
followed by credentials options related with SPECIFIED auth mode i.e.
~~~
--aws-secrets-access-key-id
--aws-secrets-secret-access-key
--aws-secrets-region
~~~
followed by filter options. Filter options also need a note similar to:
```
Note: The following/above filters are applied as AND operation.
```

**Review by usmansaleem — APPROVED** [review]
LGTM

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 3]
```diff
@@ -84,6 +84,250 @@ When
...
+
+=== "Configuration file"
+
+    ```bash
+    eth2.aws-secrets-prefixes-filter: ["prefix1","prefix2"]
+    ```
+
+Optional comma-separated list of secret name prefixes filter to apply while fetching secrets from
+AWS Secrets Manager.
+Applied as AND operation with other filters.
```
Comment (bgravenorst) [inline docs/Reference/CLI/CLI-Subcommands.md:268]: Should AND be formatted?

Applied as an `And` ...

**Review by bgravenorst — APPROVED** [review]
LGTM

**Pull request merged**